docs/modules/ROOT/pages/how-tos/migrate-to-lokistack.adoc

```diff
@@ -0,0 +1,96 @@
+= Migrate to Lokistack
```

Comment:
IMO, naming this "Switch to LokiStack" would make it clear, that no data migration should be expected.